src/components/ConfirmCard/index.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { Backdrop, Box, Button, Card, Typography } from '@mui/material';
+import { Dispatch, SetStateAction } from 'react';
+
+interface Props {
+  backdropOpen: boolean;
+  setBackdropOpen: Dispatch<SetStateAction<boolean>>;
+  confirmMessage: string;
+  handleClick: () => void;
+}
+
+export function ConfirmCard({
+  backdropOpen,
+  setBackdropOpen,
+  confirmMessage,
+  handleClick,
+}: Props) {
+  return (
+    <Backdrop
+      sx={{ color: '#fff', zIndex: 1200 }}
```

Comment:
I'd suggest to use colors from the theme everywhere we can and replace this with e.g. `color: 'white.main'`

Reply:
Should be checked if that color assignment is even necessary because the `Card` component is the visible rendered element.